static/css/tools/stat_var.scss

```diff
@@ -99,12 +120,30 @@
   box-shadow: none;
 }
 
+.start-instruction-tile {
+  background-color: #c2e7ff33;
```

Comment:
hm, this looks like a few too many characters for a color hex .. can you double check this

Reply:
This is a hexadecimal color with transparency: #c2e7ff with transparency 33 (20% opacity).

[See https://www.w3schools.com/cssref/tryit.php?filename=trycss_color_hexa](https://www.w3schools.com/cssref/css_colors_legal.php) for more info.